Changed mmtable2() to gt() in Describe < Tables dialog

anastasia-mbithe opened the pull request:
This fixes some of #8248 
@lilyclements and @rdstern This is ready for testing.

lilyclements commented:
Great! To update, @anastasia-mbithe I will have a good review of this tomorrow and get back to you then. Do you have something to work on in the meantime?

@rdstern I suggest you hold off looking at this PR until after I've reviewed?

lilyclements commented on instat/dlgSummaryTables.vb:
```diff
@@ -21,18 +21,16 @@ Public Class dlgSummaryTables
-            clsTabStyleCellTextFunction, clsTabStylePxFunction, clsTabStyleCellTitleFunction As New RFunction
+            clsTabStyleCellTextFunction, clsTabStylePxFunction, clsTabStyleCellTitleFunction, clsGtFunction As New RFunction
 
     Private clsMmtableOperator, clsSummaryOperator, clsFrequencyOperator, clsColumnOperator,
```
`clsMmtableOperator` can be removed (and any instances where you mention it)

lilyclements commented on instat/dlgSummaryTables.vb:
```diff
@@ -282,43 +273,19 @@ Public Class dlgSummaryTables
 
         clsSummariesList.SetRCommand("c")
-        clsSummariesList.AddParameter("summary_mean", Chr(34) & "summary_mean" & Chr(34), bIncludeArgumentName:=False) ' TODO decide which default(s) to use?
+        clsSummariesList.AddParameter("summary_count", Chr(34) & "summary_count" & Chr(34), bIncludeArgumentName:=False) ' TODO decide which default(s) to use?
```
Out of interest, why is this changed?

anastasia-mbithe replied:
I guess it was some errors I was experiencing while coding, and count was working. but I have tested it and am not getting the error anymore, so I have changed it back to "summary_mean"

lilyclements reviewed (changes requested):
Really nice job @anastasia-mbithe! Looks great, and works really nicely.

A few minor comments below. And in addition to these:
- [ ] The receiver(s) clear on reopen (this occurs for factors and summaries)
- [ ] If I check the "Treat Summaries as Further Factor" checkbox, then select the "Variable" rdo, then uncheck the "Treat Summaries as Further Factor" checkbox, there is nothing selected for the rdo buttons. I suggest that if "Variable" or "Summary" is ever checked (and "Treat Summaries as Further Factor" is not checked), then "Summary-Variable" should be checked? (Otherwise, clicking and unclicking the checkbox shouldn't affect the rdo). What do you think?

anastasia-mbithe commented:
@lilyclements, the first comment was fixed in #8027, but I have fixed it here again to avoid the repetitive bug in case the other pull request takes time before being merged.
I have fixed the other comments as well. Please have a look.

lloyddewit commented:
> the first comment was fixed in https://github.com/africanmathsinitiative/R-Instat/pull/8027, but I have fixed it here again to avoid the repetitive bug in case the other pull request takes time before being merged.

@rdstern I'm concerned that fixing the same issue in 2 different PRs may trigger a merge conflict. PR #8027 hopefully only needs a small regression test from you before we merge. It's been waiting for this test since 31 March so may have slipped under the radar. It would be great if we can merge PR #8027 fairly soon, thanks

rdstern reviewed (commented):
@anastasia-mbithe and @lilyclements this seems to be working ok.  Great.  I can think of many improvements I would like, but would like to merge this version, (without (mmtable2) as soon as you feel it can be a replacement.  Then it become our new baseline.  The next priority will be the simpler job of replacing mmtable2 in the one-variable and 2/3 variable cases.  Then we return to this dialogue.

lilyclements reviewed (approved):
@anastasia-mbithe this is working great. A really nice job - well done. Happy for this to be merged 🥇 

@lloyddewit over to you!

lloyddewit reviewed (approved):
@anastasia-mbithe clean, readable code as always - thanks
